src/Sentry.Extensions.Logging/SentryStructuredLogger.cs

```diff
@@ -89,6 +89,7 @@ public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Except
         };
 
         log.SetDefaultAttributes(_options, _sdk);
+        log.SetOrigin("auto.logging.dotnet_extension");
```

Comment:
question: should we have a single origin for all `Microsoft.Extensions.Logging` providers, or a unique origin per `ILoggerProvider`

Currently, we would have a single origin for `ILogger<T>`-created Structured Logs for
- `Sentry.Extensions.Logging`
- `Sentry.AspNetCore`
- `Sentry.Maui`

One the one hand, we have different SDK names (attribute `"sentry.sdk.name"`), on the other hand, I'm uncertain if we should separate the `sentry.origin` too.

E.g.
- `auto.log.aspnetcore` via `SentryAspNetCoreStructuredLoggerProvider`
- `auto.logging.maui` via `SentryMauiLoggerProvider`

Reply:
Update: `sentry.sdk.name` and `sentry.origin` are combined for filtering ... so as long as the SDK-Name is unique (which it is), a shared `Microsoft.Extensions.Logging`-style origin could be just fine, also communicating the dependencies of both "derived" `Sentry.Maui` and `Sentry.AspNetCore` to the "base" `Sentry.Extensions.Logging` SDK.